packages/block-library/src/tag-cloud/block.json

```diff
@@ -28,6 +28,10 @@
 		"largestFontSize": {
 			"type": "number",
 			"default": 22
+		},
```

Comment:
The sizes should be `strings` -  attribute (eg `largestFontSize`) with values like `8pt, 17px` etc... Check out other examples in the repo using the `UnitControl` component like `Column` and others.